### PR TITLE
Checking for law journal entry existence based only on year and entry

### PR DESCRIPTION
**Pull request by madryk**

### CODE REVIEW

File: saos-import/src/test/java/pl/edu/icm/saos/importer/commoncourt/judgment/process/LawJournalEntryCreatorTest.java [hunk 1]
```diff
@@ -36,13 +34,13 @@
     @Before
     public void before() {
         
-        ReflectionFieldSetter.setField(lawJournalEntry, "id", 114);
+        Whitebox.setInternalState(lawJournalEntry, "id", 114);
         lawJournalEntry.setYear(2011);
         lawJournalEntry.setEntry(11);
```
Comment (lukdumi) [inline saos-import/src/test/java/pl/edu/icm/saos/importer/commoncourt/judgment/process/LawJournalEntryCreatorTest.java]: setJournalNo?

**Comment by lukdumi** [comment]
The code seems to be ok.
I would also think about changing:
@Table(uniqueConstraints={@UniqueConstraint(name="law_journal_entry_unique", columnNames={"year", "journalNo", "entry"})})

In addition remember to correct the db and to change the index above manually in db (hibernate will not change it).

File: saos-import/src/test/java/pl/edu/icm/saos/importer/commoncourt/judgment/process/LawJournalEntryCreatorTest.java (hunk 1 above)
Comment (madryk) [inline saos-import/src/test/java/pl/edu/icm/saos/importer/commoncourt/judgment/process/LawJournalEntryCreatorTest.java]: :+1:

**Comment by madryk** [comment]
Ok. I will remember to change index manually

**Pull request closed without merging**